src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -333,6 +338,11 @@ export class DynamicFacet extends Component {
     this.logger.info('Show more values');
     this.dynamicFacetQueryController.increaseNumberOfValuesToRequest(additionalNumberOfValues);
     this.triggerNewQuery();
+    this.usageAnalytics.logCustomEvent<IAnalyticsFacetMeta>(
+      analyticsActionCauseList.facetShowMore,
```

Comment:
I imagine this will eventually be analyticsActionCauseList.dynamicFacetShowMore,

Reply:
After talking with @olamothe, we will add a dynamicFacetShowMore event when the need arise